.github/ISSUE_TEMPLATE/copyright-edit-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Copyright Edit Request
```

Comment:
This wouldn't be considered copyright edit, that's a legal thing. Should be called "Content Edit Request"